cursorless-talon/src/fallback.py

```diff
@@ -0,0 +1,107 @@
+from typing import Callable
+
+from talon import actions
+
```

Comment:
import client version from wherever it's defined here in talon, and add assertion here that it is equal to 7, and add a comment. That will ensure we remember to update this stuff if we change request payload shape

Reply:
done